src/sota_tools/authenticate.cc

```diff
@@ -8,17 +8,17 @@ using std::string;
 
 int authenticate(const string &cacerts, const ServerCredentials &creds, TreehubServer &treehub) {
   switch (creds.GetMethod()) {
-    case AUTH_BASIC: {
+    case AuthMethod::BASIC: {
```

Comment:
Sorry to keep bugging, but can we rename these all-caps ones, too?

Reply:
So, `AuthMethod::kBasic`?

Reply:
Yeah, I think that's best.